Pipfile

```diff
@@ -87,7 +87,7 @@ service_factory = ">=0.1.5"
 pytest-watch = "*"
 rope = "*"
 "autopep8" = "<=1.4.0"
-pre-commit= "==1.11.1"
+pre-commit= "==1.15.1"
```

Comment:
1.11.1 has an issue with the check large files command. Updated to a newer version to fix pre-commit checks.